README.md

```diff
@@ -90,6 +90,52 @@ Then restart Chrome for these changes to take effect.
 > [!NOTE]
 > For more information about Chrome Built-in AI: https://developer.chrome.com/docs/ai/get-started
 
+## Using LiteLLM Proxy
```

Comment:
Added a bit of documentation to cover the use case mentioned in https://github.com/jupyterlite/ai/issues/144